assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -146,6 +177,8 @@ function DashboardSummaryWidget( { Widget, WidgetReportZero, WidgetReportError }
 						title={ __( 'RPM', 'google-site-kit' ) }
 						datapoint={ period.totals[ 1 ] }
 						datapointUnit={ currencyCode }
+						change={ Number( change.rpm ) }
```

Comment:
The `Number` cast shouldn't be necessary anymore as the value should already be a number.